standard/expressions.md

```diff
@@ -968,6 +968,16 @@ For a function member that includes a parameter array, if the function member is
   - for a fixed value parameter or a value parameter created by the expansion, an implicit conversion ([§10.2](conversions.md#102-implicit-conversions)) exists from the argument expression to the type of the corresponding parameter, or
   - for a `ref` or `out` parameter, the type of the argument expression is identical to the type of the corresponding parameter.
 
+In the case of a method group
+
+- If the group contains both instance and static members, instance members invoked without an instance receiver or context are not applicable.
```

Comment:
Should this be about the group containing both *applicable* instance and static members? (I'm wondering whether this algorithm is order-independent and so raising the question)